src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js

```diff
@@ -94,6 +94,12 @@ const ParentIconStyle = styled.span`
   }
 `;
 
+const StyledIcon = styled(Icon)`
+  & > * {
```

Comment:
Isn't `*` too generic? cannot be selected by `svg`?

Reply:
That works as well - changed it